flutter/test/load_android_image_list_test.dart

```diff
@@ -125,3 +132,21 @@ SentryEvent getEvent({bool symbolicated = false}) {
   );
   return SentryEvent(exception: ex);
 }
+
+class Fixture {
+  Fixture() {
+    channel = MethodChannel('sentry_flutter');
+    options = SentryFlutterOptions()..dsn = fakeDsn;
+    hub = Hub(options);
```

Comment:
every other fixture initializes the fields directly, why is it within a ctor and making them `late`, is there any benefit in Dart?

Reply:
I can't initialize `Hub` in a field because it needs `SentryOptions`. I think all other properties could be initialized in a field.